apps/src/templates/certificates/Certificate.jsx

```diff
@@ -65,7 +65,15 @@ function Certificate(props) {
       course: props.tutorial,
       donor
     };
-    return btoa(reEncodeNonLatin1(JSON.stringify(data)));
+    const asciiData = btoa(reEncodeNonLatin1(JSON.stringify(data)));
+    const urlSafeData = asciiData
+      .replace(/\+/g, '-')
```

Comment:
I think you could use `replaceAll()` which would avoid the need to use a regex pattern with the g modifier. It might read a bit easier?

Example:

`asciiData.replaceAll('+', '-').replaceAll('/', '_')...`

Reply:
Actually this is causing issues, see [this comment](https://github.com/code-dot-org/code-dot-org/pull/51770#discussion_r1187798710)